app/Console/Commands/RoulettePlay.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Contracts\Cache\Repository as CacheRepository;
+
+class RoulettePlay extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'roulette:play';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Play russian roulette game.';
+
+    private CacheRepository $cacheRepository;
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct(CacheRepository $cacheRepository)
+    {
+        parent::__construct();
+        $this->cacheRepository = $cacheRepository;
+    }
+
+    /**
+     * Execute the console command.
+     *
+     * @return int
+     */
+    public function handle()
+    {
+        $name = $this->ask(
+            "Only the brave will play this game. Please, name yourself !"
+        );
+        do {
+            $bulletInChamber = rand(1,6);
+            $playerGotChamber = rand(1,6);
+
+            $this->info("Bullet in chamber: " . $bulletInChamber);
+            $this->info("Player got chamber: " . $playerGotChamber);
+
+
+            if($bulletInChamber === $playerGotChamber)
+            {
+                $this->info("Congrats! Back to zero leader score. Reason: You died !");
+                $statistics = $this->cacheRepository->get('Roulette results', []);
+                $statistics[$name] = 0;
+
+                $this->cacheRepository->set('Roulette results', $statistics, 86400);
+
+                $killstreak = $this->cacheRepository->get('Kill-streak', 0);
+                $killstreak++;
+                $this->cacheRepository->set('Kill-streak', $killstreak, 86400);
```

Comment:
Please keep in statistic, also number of times specific number was chosen. and keep it in statistics command.

Reply:
Now showing kill-streak on statistics command.
Now keeping in cache Cursed Numbers at which players loose the game. Also showing them in statistics.